Mobile/starter_project_dartsmiths/lib/features/onboarding/onboardingUpperLayout.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+class LandingPageUpperImage extends StatelessWidget {
+  const LandingPageUpperImage({super.key});
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        Padding(
+          padding: const EdgeInsets.only(right: 10.0),
+          child: Container(

```

Comment:
same as the above containers.

Reply:
I am using decorations. SizedBox can't help me for such like styles.